lib/Rex/Commands/Iptables.pm

```diff
@@ -182,8 +203,17 @@ Redirect $in_port to another local port.
 =cut
 
 sub redirect_port {
-  my ( $in_port, $option ) = @_;
+  my (@params) = @_;
+  my $ip_version = _get_ip_version( \@params );
+  if ( $ip_version == -6 ) {
+    my $iptables_version = _iptables_version();
+    if ( $iptables_version < 1_004_018 ) {
+      Rex::Logger::info("IPtables < v1.4.18 doesn't support NAT for IPv6");
+      die("IPtables < v1.4.18 doesn't support NAT for IPv6");
```

Comment:
I think the official name is `iptables` (vs. `IPtables`), based on a quick glance at their [homepage](http://www.netfilter.org/projects/iptables/). If you agree, can you update the messages here, please?